activerecord/lib/active_record/connection_adapters/abstract_mysql_adapter.rb

```diff
@@ -648,6 +649,21 @@ def check_version # :nodoc:
         end
       end
 
+      def strip_whitespace_characters(expression)
+        # The Rubocop statements are required here because the expression
+        # may contain strings of these characters, _not_ the escaped versions
+        # that using double quotes would produce.
+        #
+        # rubocop:disable Performance/StringReplacement
+        # rubocop:disable Style/StringLiterals
+        expression = expression.gsub('\n', "")
+        expression = expression.gsub('\\', "")
+        expression = expression.gsub(/\s{2,}/, " ")
+        # rubocop:enable Style/StringLiterals
+        # rubocop:enable Performance/StringReplacement
```

Comment:
We don't generally do rubocop enable/disable in the files because it pollutes them. We instead add them to the exclude list. 

This method should also be moved to the private namespace, we don't want applications to call this method.

Reply:
Will tweak that this afternoon, thanks @eileencodes.

Reply:
Tweaks made!

Reply:
I had to add a couple of extra files to the exclusions for `Performance/StringReplacement`, when I excluded the file I edited there were other changes that Rubocop flagged. Let me know if those need to be removed again.